spec/Board_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rspec'
+require'./lib/Game'
+require "./lib/Player"
+require"./lib/Board"
+describe Board do 
+   describe '#add_piece' do
+      it 'returns true' do
```

Comment:
instead of `return true`, you need to add some relevant description about what test is doing. Let's say here it should be `it 'checks empty slot' do`